plugins/services/src/js/components/dsl/FuzzyTextDSLSection.js

```diff
@@ -0,0 +1,48 @@
+import React, {PropTypes} from 'react';
+import DSLCombinerTypes from '../../../../../../src/js/constants/DSLCombinerTypes';
```

Comment:
We have a coding style rule to separate node_module imports and application imports with a blank line. Thanks!
Also got the same issue in other files.